### PR TITLE
Proper deprecate MAV_ACTION

### PR DESCRIPTION
**Pull request by amilcarlucas**

### CODE REVIEW

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -232,8 +232,8 @@
         <description>Mission failure.</description>
       </entry>
     </enum>
-    <!-- WARNING: MAV_ACTION Enum is no longer supported - has been removed. Please use MAV_CMD -->
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml]: Yes, this can be deleted.

File: message_definitions/v1.0/common.xml [hunk 2]
```diff
@@ -232,8 +232,8 @@
...
     <enum name="MAV_MODE_FLAG">
+      <deprecated since="2012-02" replaced_by="MAV_CMD_DO_SET_MODE">No longer supported - has been removed.</deprecated>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml]: I think this deprecation is probably wrong. It nothing to do with MAV_ACTION.
MAV_CMD_DO_SET_MODE replaced SET_MODE, but a message can't replace a flag "MAV_MODE_FLAG". This flag is also used in a number of messages, which are not deprecated.

File: message_definitions/v1.0/common.xml (hunk 1 above)
Comment (amilcarlucas) [reply message_definitions/v1.0/common.xml]: Looks like only this leftover needs to be removed

**Comment by hamishwillee** [comment]
Thanks @amilcarlucas ! Merged :-)

**Pull request merged**